lib/kubeclient/common.rb

```diff
@@ -39,6 +39,12 @@ module ClientMixin
       'fieldSelector' => :field_selector
     }.freeze
 
+    DELETE_ARGUMENTS = {
+      'gracePeriodSeconds' => :grace_period_seconds,
+      'orphanDependents'   => :orphan_dependents,
```

Comment:
true.  I suggest focusing on `1.7. Shall I remove `orphanDependents` and we don't have this issue?